guides/v2.3/inventory/source-selection-algorithms.md

```diff
@@ -0,0 +1,174 @@
+---
+group: inventory
+title: Source selection algorithms
+---
+
+The **Source Selection Algorithm (SSA)** recommends how to fulfill partial and full shipments. The merchant decides which business needs take precedence when deciding which shipping method to use:
+
+* Should the products be delivered from the sources designated as having the highest priority? 
+* Should the total shipment cost be the primary factor in choosing a shipment method?
+* Should the shipments originate from the closest source?
+* Should the fastest shipping method be used, even if it's not the cheapest?
+
+Magento 2.3 provides the priority-based SSA only. Third party developers can create their own algorithms to 
+help merchants decide which shipping option best meets their needs.
+
+Magento does not enforce or save the results of SSA recommendations. The recommendations reflect conditions at the moment when the algorithm runs, but conditions change over time. For example, the amount of in-stock products will always fluctuate, and shipping costs might change. The merchant can also modify the recommendations by adjusting quantities for deduction or even by re-assigning the shipment's sources of origin.  
+
+## The priority algorithm
+
+Inventory Management defines the priority algorithm, which recommends delivering products from sources having the highest priority. The `SourceSelectionServiceInterface` accepts an `InventoryRequestInterface` object, which in turn contains the stock ID and a list of items to be shipped. Each item contains only the SKU and quantity. Other potentially relevant data, such as shipping address, is not included, because the priority algorithm does not need it. 
+
+Additional input data might be needed for more sophisticated algorithms. For example, an algorithm that calculates distance would need the shipping address. that's why `InventoryRequestInterface` implements `ExtensibleDataInterface` interfaces which could be extended with custom input parameters
```

Comment:
```suggestion
Additional input data might be needed for more sophisticated algorithms. For example, an algorithm that calculates distance would need the shipping address and all data entered for the source (GPS or full address). That's why `InventoryRequestInterface` implements `ExtensibleDataInterface` interfaces, which can be extended with custom input parameters.
```

Reply:
I mentioned a complete source cause address/GPS is not required. They only need country + zip. The other bits may be needed/useful for a distance algorithm...?